.gitignore

```diff
@@ -6,6 +6,4 @@ __pycache__/
 
 # TS files
 dist/
-*.tsbuildinfo
-
-.DS_Store
+*.tsbuildinfo
```

Comment:
I would tend to leave .gitignore as is to avoid peaople are commiting .DS_Store files on Mac